rsknode/Dockerfile

```diff
@@ -24,15 +24,17 @@ RUN ./gradlew --no-daemon clean build -x test
 FROM openjdk:8-jdk-slim-buster
 
 ARG rskj_codename=PAPYRUS
-ARG rskj_version=2.1.0
+ARG rskj_version=2.2.0
 
 WORKDIR /usr/local/rskj
 COPY --from=build /code/rskj/rskj-core/build/libs/rskj-core-${rskj_version}-${rskj_codename}-all.jar ./rsk.jar
 COPY node.conf ./node.conf
 COPY logback.xml ./logback.xml
 
+RUN mkdir -p /root/.rsk/regtest/database
+RUN chmod -R 777 /root/.rsk
+
 ENTRYPOINT ["java", \
     "-Drsk.conf.file=./node.conf", \
     "-Dlogback.configurationFile=./logback.xml", \
-    "-Ddatabase.dir=./db", \
```

Comment:
Can you explain the reason for this changes?

Reply:
It didn't work, as it could not find the directory

Also, the version 2.1.0 didn't show the error messages, I I neeeded them fot debugging purpouses. But if you prefer i can rollback that change and set it to 2.1.0 again